Tests/Web/html/tests/usersystem_tests.js

```diff
@@ -91,13 +91,19 @@ test('UserSystemTests', 'FalseAgeVerificationLoginTest', async function() {
 
     // Log in with false age verification
     {
-        await logIn(userSystem, DEFAULT_LOGIN_EMAIL, DEFAULT_LOGIN_PASSWORD, false,Services.EResultCode.Failed,Services.EResultBaseFailureReason.None);
+        await logIn(userSystem, DEFAULT_LOGIN_EMAIL, DEFAULT_LOGIN_PASSWORD, false, Services.EResultCode.Failed, 1);
+
+    }
+
+    // Log in with null age verification
+    {
+        await logIn(userSystem, DEFAULT_LOGIN_EMAIL, DEFAULT_LOGIN_PASSWORD, null, Services.EResultCode.Failed, 1);
```

Comment:
Perhaps a comment here similar to what you've got with the C++ tests? 1 stands for age not verified type thing

Reply:
this has changed since please recheck